src/fal/run_scripts.py

```diff
@@ -53,15 +85,19 @@ def run_scripts(scripts: List[FalScript], faldbt: FalDbt) -> List[bool]:
         pool: Pool = pool
         try:
             scripts_with_project = map(lambda script: (script, faldbt), scripts)
-            results = pool.starmap(_prepare_exec_script, scripts_with_project)
+            results = pool.starmap(run_script, scripts_with_project)
```

Comment:
Eventually we will remove this threading part, right?

Reply:
Yes!